src/CribblyBackend/Controllers/GameController.cs

```diff
@@ -35,6 +35,24 @@ public async Task<IActionResult> GetById(int id)
             logger.Information("Request for game {id} returned no results", id);
             return NotFound();
         }
+        
+        /// <summary>
+        /// GetByTeamId fetches all games associated with a given TeamId.
+        /// </summary>
+        /// <param name="id">The id of the Team for which to get all games</param>
+        /// <returns></returns>
+        [HttpGet]
+        [Route("GetByTeamId")]
+        public async Task<IActionResult> GetByTeamId(int id)
```

Comment:
Would be more intuitive to instead define this endpoint in the teams controller? Then a client could request a get from `/teams/12345/games`